api/base/filters.py

```diff
@@ -112,6 +113,26 @@ def remove_invalid_fields(self, queryset, fields, view, request):
         return valid_fields
 
 
+class ElasticOSFOrderingFilter(OSFOrderingFilter):
```

Comment:
What's this for?

Reply:
The search endpoints use ES to create their querysets, this sorts querysets from Elasticsearch which aren't Django querysets but just dicts.